x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -123,7 +123,7 @@ setup:
   - match: {esql.functions.coalesce: $functions_coalesce}
   - gt: {esql.functions.categorize: $functions_categorize}
   # Testing for the entire function set isn't feasible, so we just check that we return the correct count as an approximation.
-  - length: {esql.functions: 141} # check the "sister" test below for a likely update to the same esql.functions length check
+  - length: {esql.functions: 142} # check the "sister" test below for a likely update to the same esql.functions length check
```

Comment:
Gotta add one to the count below or the release tests with fail. This is an annoying test....